src/layouts/Account-HEDA Organization Layout.layout

```diff
@@ -187,7 +193,7 @@
     <showRunAssignmentRulesCheckbox>false</showRunAssignmentRulesCheckbox>
     <showSubmitAndAttachButton>false</showSubmitAndAttachButton>
     <summaryLayout>
-        <masterLabel>00h1D000000dfPU</masterLabel>
```

Comment:
Very curious about the masterLabel here

Reply:
This is a good question. I've seen this happen before when doing a retrieve from the XML.

Reply:
@xinnanlisf -- This is what I found here: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_layouts.htm?search_text=summaryLayout -- I don't think this affects anything and seems to be Classic related? It's not super clear...